travis/prepare/prepare-javascript

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Configuring display
+sh -e /etc/init.d/xvfb start > /dev/null 2>&1
+export DISPLAY=:99
+
+# Installing google-chrome
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb > /dev/null 2>&1
+sudo dpkg -i google-chrome-stable_current_amd64.deb > /dev/null 2>&1
```

Comment:
Primo, Firefox is much slower than Chrome (about 2-3 times slower).
Secundo, Firefox has a random issues with some pages, sometimes it scrolls down the page and there's no possibility to scroll up; when it tries to do something with element that isn't visible and accessable for that moment, it throws an exception and test fails.